financial-connections/src/main/java/com/stripe/android/financialconnections/repository/FinancialConnectionsErrorRepository.kt

```diff
@@ -1,30 +1,27 @@
 package com.stripe.android.financialconnections.repository
 
-import com.airbnb.mvrx.ExperimentalMavericksApi
-import com.airbnb.mvrx.MavericksRepository
 import com.airbnb.mvrx.MavericksState
-import com.stripe.android.financialconnections.BuildConfig
-import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.update
 
-@OptIn(ExperimentalMavericksApi::class)
-internal class FinancialConnectionsErrorRepository(
-    coroutineScope: CoroutineScope
-) : MavericksRepository<FinancialConnectionsErrorRepository.State>(
-    initialState = State(),
-    coroutineScope = coroutineScope,
-    performCorrectnessValidations = BuildConfig.DEBUG,
-) {
+internal class FinancialConnectionsErrorRepository {
 
-    suspend fun get() = awaitState().error
+    private val state = MutableStateFlow(State())
```

Comment:
Why do we need a flow here?

Reply:
https://github.com/stripe/stripe-android/pull/8154/commits/38c29bc8dc5fb45b0fab966180605b70dc49de6f